tools/gcp/delete

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 
-gcloud compute instances delete \
-    $@ \
-    --quiet
\ No newline at end of file
+gcloud compute instances delete $@ --quiet
```

Comment:
we should loop here like in `create`

Reply:
You can pass multiple instances here and it will delete them all. Only reason we're not doing that for `create` is because we need to pass different args.